### ISSUE TITLE
Nice theme

### ISSUE THREAD

**Issue by DutchPete** [exloseur3d/nier-theme#3]
Hi, well done with this theme, mes félicitations!
As a geologist, perhaps 1 day you can make a theme showing the origin of obsidian - suggestion from a fellow geologist 😀.

**Comment by exloseur3d** [comment]
Haha, sure! I actually would like to make a theme based on one of my favorite minerals, Olivine.

**Comment by DutchPete** [comment]
![image](https://github.com/user-attachments/assets/815e68dd-2f9a-4af7-906d-b23a869dab96)

**Comment by justinmeader** [comment]
@exloseur3d I don't see the theme in settings when I extract it, which might be my issue - but it also doesn't seem to be available under the community themes in settings. Was it delisted?

**Comment by exloseur3d** [comment]
Hello, because the theme is only in dark theme, you have to select dark mode in appearance settings in Obsidian. Then, you can go to manage, and you should be able to find it and install it.

**Comment by exloseur3d** [comment]
@justinmeader 
Let me know if this fixes your problem.

**Comment by justinmeader** [comment]
@exloseur3d I feel dumb, didn't realize it would totally filter out dark themes if I had it set to Auto. Thank you!

**Comment by gentlescribe** [comment]
Hello, I really love the Nier theme but could you perhaps make a dark mode version or Change the font color of this bar, to make it more visible 

![Image](https://github.com/user-attachments/assets/52387a0e-91eb-4bd6-87ac-8aa06859e455)

**Comment by exloseur3d** [comment]
@gentlescribe Hey, the update is already online. It was an easy fix. I hope you like it.

**Comment by gentlescribe** [comment]
@exloseur3d  Thank you very much, yes I do like it😊

**Comment by gentlescribe** [comment]
Yes thank you, I've updated the theme and it's really amazing. Thank you
for the fix
